test/html_test.exs

```diff
@@ -68,37 +68,4 @@ defmodule ExCoveralls.HtmlTest do
     assert(size == @file_size)
   end
 
-  test_with_mock "Exit status code is 1 when actual coverage does not reach the minimum",
```

Comment:
@vereis Very sorry not being responsive. Could help me understand these removed portion of test codes? The intention is the relevant conditions are covered in other test cases? (just wanted to confirm the exiting behaviors can be maintained if the new options is not specified).

Reply:
@vereis it sounds like we might actually want these removed tests? Possibly adding them to the other coverage types as well?